analyzer/src/funTest/kotlin/managers/BabelFunTest.kt

```diff
@@ -22,18 +22,17 @@ package org.ossreviewtoolkit.analyzer.managers
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.matchers.shouldBe
 
-import java.io.File
-
 import org.ossreviewtoolkit.downloader.VersionControlSystem
 import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.utils.ort.normalizeVcsUrl
 import org.ossreviewtoolkit.utils.test.USER_DIR
+import org.ossreviewtoolkit.utils.test.getAssetFile
 import org.ossreviewtoolkit.utils.test.patchActualResult
 import org.ossreviewtoolkit.utils.test.patchExpectedResult
 
 class BabelFunTest : WordSpec({
-    val projectDir = File("src/funTest/assets/projects/synthetic/npm-babel").absoluteFile
+    val projectDir = getAssetFile("projects/synthetic/npm-babel").absoluteFile
```

Comment:
Do you think it makes sense to change the implementation of `getAssetFile()` to call `absoluteFile` so we could drop these here?

Reply:
I also stumbled across this `absoluteFile` thing. I found that `projectDir` does not necessarily need to be absolute.
The need for it only comes from `definitionFile`. I started fixing that, but then decided to first eliminate code redundancy, e.g. this bit of code is in almost any funTest. I have this on my TODO list. Once the redundancy is removed, there would be no need to change the implementation for `getAssetFile()` anymore. I'd propose to go this path, if you agree it makes sense...?!